charts/stable/tdarr/templates/common.yaml

```diff
@@ -19,6 +19,10 @@ additionalContainers:
         value: "{{ .Values.service.main.ports.server.port }}"
       - name: ffmpegPath
         value: {{ default "" .Values.env.ffmpegPath }}
+      - name: PUID
+        value: {{ default "1000" .Values.node.PUID }}
+      - name: PGID
+        value: {{ default "1000" .Values.node.PGID }}
```

Comment:
These defaults aren't really needed I think, since both already have a default set in the `values.yaml` file.

Also, you may need to replace `.Values.node.PUID` with `(.Values.node.PUID | quote)` (this also goes for PGID)

Reply:
Is it correct now?

Reply:
@bjw-s is it correct now?